### ISSUE TITLE
Planejamento da aquisição de equipamentos para Sala Interativas Remotas

### ISSUE THREAD

**Issue by lcaimi** [practice-uffs/programa#102]
Para registro das atividades já finalizadas do processo de compras de equipamentos para Salas Interativas Remotas.

**Comment by lcaimi** [comment]
Pregão agendado: 07/10

[https://www.uffs.edu.br/UFFS/atos-normativos/pregao/sucl/2020-0032](https://www.uffs.edu.br/UFFS/atos-normativos/pregao/sucl/2020-0032)

**Comment by Dovyski** [comment]
Apenas para confirmar, nós podemos fazer a divulgação desse link para todas as empresas em nosso círculo para que eles estejam no pregão, certo, @lcaimi ?

**Comment by lcaimi** [comment]
Sim. Todos podem auxiliar na divulgação utilizando o link acima.
